lychee-lib/src/types/response.rs

```diff
@@ -64,7 +64,7 @@ impl Display for ResponseBody {
             }
             Status::Timeout(Some(code)) => format!(" [{}]", code),
             Status::Error(e) => format!(" ({})", e),
-            _ => "".to_owned(),
+            _ => "".into(),
```

Comment:
No.
When converting a &str to String, I deliberately use `.to_owned()` to indicate the memory allocation.

Reply:
This doesn't allocate; I tough about using `String::new()` but decided that this is clearer.

Reply:
The whole code base consistently use `str::to_owned()` for this type of conversion.
When you refactor you need to take the overall coding style and convention into account.

Reply:
Sure, I'll revert it